src/js/profile/mobile/widget/Calendar.js

```diff
@@ -175,10 +186,29 @@
 					today = new Date(todayYear, todayMonth - 1);
 					prototype._buildCalendar(calendarElement);
 				} else {
-					event.preventDefault();
+					if (event.path.length === 12) {
```

Comment:
What's mean the "event.path.length===12"?  Is it month??

Reply:
I was worried about this part.
Click to separate.
There are five clickable parts.
1. arrow = 11
2. title = 11
3. one week = 11
4. day = 12
5. empty area = 10

I found the difference between the 5 elements.
event.path.length is different. event.path.length increases as the tag is included.
I had to get to click only day, so I wrote it in the code.

Reply:
It won't work ... you can't know how many levels of nesting will occur. You have to check css class.

Reply:
@singa2000 @TomaszLukawskiSam 

I add ">div" selector to css.
Clicking on a date by ">div" or clicking on the area around the date creates the same selection.